src/Nri/Ui/Block/V4.elm

```diff
@@ -304,6 +329,20 @@ blankWithId =
     BlankWithId
 
 
+{-| Wraps a group of content in a `strong` tag
+-}
+bold : List (Content msg) -> Content msg
+bold =
+    Markdown Bold
+
+
+{-| Wraps a group of content in a `em` tag
+-}
+italic : List (Content msg) -> Content msg
+italic =
+    Markdown Italic
+
+
```

Comment:
I thought perhaps that `strong` and `emphasis` might be better names semantically, but was worried about `emphasis` causing confusion with how we also us emphasis to describe display elements.